device/dhcp/servers/v6/dhcpv6server.yaml

```diff
@@ -0,0 +1,212 @@
+components:
+  schemas:
+    DhcpServer.V6:
+      description: >-
+        Configuration for emulated DHCPv6 Server.
+      type: object
+      x-status:
+        status: under_review
+      required: [name,ipv6_name,leases]
+      properties:
+        name:
+          x-include: ../../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        ipv6_name:
+          description: >-
+            The unique name of the IPv6 on which DHCPv6 server will run.
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ipv6/properties/name'
+          x-field-uid: 2
+        rapid_commit:
```

Comment:
in case of v4 , rapid_commit is omitted in the first implementation.

Reply:
Currently keeping rapid commit, if dut enables we will not have the option to support it